share/ci/scripts/linux/dnf/install_doxygen.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the OpenColorIO Project.
+
+set -ex
+
+DOXYGEN_VERSION="$1"
+
+if ! command -v doxygen >/dev/null; then
+    if command -v dnf >/dev/null; then
+        if [ "$DOXYGEN_VERSION" == "latest" ]; then
```

Comment:
Following the fix to the apt/install_doxygen.sh file, I think this should be a singe "=" too.
```suggestion
        if [ "$DOXYGEN_VERSION" = "latest" ]; then
```